src/main/java/swervelib/SwerveDrive.java

```diff
@@ -1042,12 +1023,7 @@ public void setGyroOffset(Rotation3d offset) {
   public void addVisionMeasurement(Pose2d robotPose, double timestamp) {
     odometryLock.lock();
     swerveDrivePoseEstimator.addVisionMeasurement(new Pose2d(robotPose.getX(), robotPose.getY(), getYaw()), timestamp);
-    // Pose2d newOdometry = new Pose2d(swerveDrivePoseEstimator.getEstimatedPosition().getTranslation(),
-    //     robotPose.getRotation());
     odometryLock.unlock();
-
-    // setGyroOffset(new Rotation3d(0, 0, robotPose.getRotation().getRadians()));
-    // resetOdometry(newOdometry);
```

Comment:
The resetOdometry call was already removed by @AlexanderGardiner , so the main goal of this PR was already accomplished by that. Still, the other parts of this PR are useful.